proto2ros/msg/Value.msg

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
```

Comment:
> Is there a protobuf concept that it maps to?

There is. See https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/struct.proto#L56-L78.

> One specific confusion: a list of float64. When is it
> - a Value.msg with the list_value set?
> - a List.msg? where each Value entry has the float64 set.

You could technically do both. When put in context, as values in a Struct.msg, it can only be the former.

Reply:
Ah, okay! A link to https://protobuf.dev/reference/protobuf/google.protobuf/#value would be great.

Reply:
Done in https://github.com/bdaiinstitute/ros_utilities/pull/44/commits/6b9f4dd79bb2a41bf30d44863d1d0adcd1607275.